src/main/resources/jenkins/plugins/http_request/HttpRequest/config.jelly

```diff
@@ -37,21 +38,7 @@
             <f:select />
         </f:entry>
         <f:entry title="Custom headers">
-            <f:repeatable field="customHeaders">
-                <table width="100%">
-                    <f:entry title="Header" field="name">
-                        <f:textbox/>
-                    </f:entry>
-                    <f:entry title="Value" field="value">
-                        <f:textbox/>
-                    </f:entry>
-                    <f:entry>
-                        <div align="right">
-                            <f:repeatableDeleteButton/>
-                        </div>
-                    </f:entry>
-                </table>
-            </f:repeatable>
+            <f:repeatableProperty field="customHeaders" />
```

Comment:
When I move the `repeatableDeleteButton` inside the `repeatableProperty` like so:
```
         <f:entry title="Custom headers">
-            <f:repeatableProperty field="customHeaders" />
+            <f:repeatableProperty field="customHeaders">
+                <f:entry>
+                    <div align="right">
+                        <f:repeatableDeleteButton/>
+                    </div>
+                </f:entry>
+            </f:repeatableProperty>
         </f:entry>
```
And when I also remove the `repeatableDeleteButton` from its current location inside the `NameValuePair/config.jelly`, the delete button disappears all together.